### ISSUE TITLE
The head and some arms joints cannot be restarted when pushing the fault button

### ISSUE THREAD

**Issue by S-Dafarra** [robotology/icub-firmware#83]
When pushing the fault on ``iCubGenova02`` or ``iCubGenova04`` some joints cannot run until a full reboot. In particular this happens all the time for the head joints and sometimes for the arms joints starting from the ``wrist_pronosup``.
Here a log 
[log_icub-head_yarprobotinterface_10797.txt](https://github.com/robotology/icub-firmware/files/1706773/log_icub-head_yarprobotinterface_10797.txt)

**Comment by S-Dafarra** [comment]
cc @gabrielenava @marcoaccame @valegagge

**Comment by marcoaccame** [comment]
Hi @S-Dafarra, thanks for the notification. We looked at the problem straight away last week and we have already tested a solution on the green robot. 

The source code is already committed (https://github.com/robotology/icub-firmware/commit/f3c3b94470eceecffbdc9b4e81b93e1222837e39) and the binary will be committed within the week. We are in the process of merging devel in master and hence, we prefer to pay attention to a new binary release.

I will alert you when everything is ready.
